src/main/resources/predicate_snap_functions_axioms.smt2

```diff
@@ -32,8 +32,8 @@
 
 (assert (forall ((s $Snap) (pm $PPM)) (!
     ($Perm.isValidVar ($PSF.perm_$PRD$ pm s))
-    :pattern ($PSF.perm_$PRD$ pm s))))
+    :pattern (($PSF.perm_$PRD$ pm s))))) ; ensure :pattern ((f x)) instead of pattern (f x) since pattern must a be a list.
```

Comment:
Maybe this comment need not be in all the places this change was done. For future reference it is good to note that this change was made e.g. in the PR description.

Reply:
Fixed by commit 60de52011848e255b37e077b3a4024f8bfd28a05. The comments are moved into the commit message.